docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -3379,6 +3379,55 @@ def change
 end
 ----
 
+[source,ruby]
+----
+# remove_columns
+
+# bad
+def change
+  remove_columns :users, :name, :email
+end
+
+# good
+def change
+  reversible do |dir|
+    dir.up do
+      remove_columns :users, :name, :email
+    end
+
+    dir.down do
+      add_column :users, :name, :string
+      add_column :users, :name, :email
```

Comment:
Is this a typo?

```suggestion
      add_column :users, :email, :string
```

Reply:
Updated, but I can't regenerate docs now:

```
bundle exec rake default                        
rake aborted!
LoadError: cannot load such file -- rubocop/cops_documentation_generator
tasks/cops_documentation.rake:6:in `require'
tasks/cops_documentation.rake:6:in `<top (required)>'
/Users/philcoggins/Workbench/rubocop-rails/Rakefile:6:in `load'
/Users/philcoggins/Workbench/rubocop-rails/Rakefile:6:in `block in <top (required)>'
/Users/philcoggins/Workbench/rubocop-rails/Rakefile:6:in `each'
/Users/philcoggins/Workbench/rubocop-rails/Rakefile:6:in `<top (required)>'
/Users/philcoggins/.rbenv/versions/2.6.6/bin/bundle:23:in `load'
/Users/philcoggins/.rbenv/versions/2.6.6/bin/bundle:23:in `<main>'
```

Reply:
> Updated, but I can't regenerate docs now:

Can you try running `bundle update`?

Reply:
Anyway, the doc has been updated. I'm going to merge this PR. Thank you for your contribution!

Reply:
Ah, I overlooked that the document generation hasn't finished yet 💦 I committed below:
https://github.com/rubocop-hq/rubocop-rails/commit/d8f0f0b